src/main/rx/rx.c

```diff
@@ -502,6 +502,7 @@ bool rxUpdateCheck(timeUs_t currentTimeUs, timeDelta_t currentDeltaTimeUs)
 
         break;
 #endif
+    case RX_PROVIDER_UDP:
```

Comment:
Add UDP receiver.

Reply:
(please move it to last line, to match enum order)